clerk/backend/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "backend"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4.4.1"
+clerk-rs = "0.2.3"
+dotenv = "0.15.0"
```

Comment:
For loading the secret key on the Rust side, we want to use shuttle-secrets and a Secrets.toml.example file instead of dotenv and .env file

Reply:
Sure will use them